### PR TITLE
Adds Jenkinsfile to execute a job

### PR DESCRIPTION
**Pull request by Karm**
It is the proverbial "Minimal Viable Product" :smiley: 
It works; running ATM.

### CODE REVIEW

**Comment by Karm** [comment]
@Ladicek It was @rsvoboda's idea :see_no_evil:

**Comment by Ladicek** [comment]
Hard to argue with the almighty @rsvoboda :godmode: :grin:

**Comment by Karm** [comment]
```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  08:48 min
[INFO] Finished at: 2020-02-14T04:31:15-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.22.1:verify (default) on project configmap: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/hudson/workspace/workspace/quarkus-openshift-test-suite/configmap/target/failsafe-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :configmap
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] junit
Recording test results
No test report files were found. Configuration error?
Error when executing always post condition:
hudson.AbortException: No test report files were found. Configuration error?
	at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:154)
	at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:118)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2917)
```

Needs a better exp in junit post build step IMHO....

**Comment by Ladicek** [comment]
I think I've always used something like `**/target/*-reports/*.xml`

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    stages {
+        stage('Run tests') {
+            steps {
+                sh "./mvnw clean verify -Dversion.quarkus=${QUARKUS_VERSION}"
+            }
+        }
+    }
+    post {
+        always {
+            junit '**/**/TEST-*xml'
```
Comment (Ladicek) [inline Jenkinsfile]: `**/**/TEST-*xml` is equivalent to `**/TEST-*xml` (and ugly :-) )

**Comment by Karm** [comment]
@Ladicek Ready for merge.

**Comment by Ladicek** [comment]
Thanks!

**Pull request merged**